client/blocks/nps-survey/index.jsx

```diff
@@ -71,45 +75,51 @@ class NpsSurvey extends Component {
 		const shouldDisableControls = this.state.isSubmitting || this.state.isSubmitted;
 
 		return (
-			<div className={ className }>
+			<Card className={ className }>
 				<div className="nps-survey__question-screen">
-					<div>How likely is it that you would recommend WordPress.com to your friends, family, or colleagues?</div>
-					<div>
+					<div className="nps-survey__question">
+						{ translate( 'How likely are you to recommend WordPress.com to your friends, family, or colleagues?' ) }
+					</div>
+					<div className="nps-survey__recommendation-select-wrapper">
 						<RecommendationSelect
 							value={ this.state.recommendationValue }
 							disabled={ shouldDisableControls }
 							onChange={ this.handleRecommendationSelectChange }
 						/>
 					</div>
-					<div>
+					<div className="nps-survey__buttons">
 						<Button primary
 							className="nps-survey__finish-button"
 							disabled={ shouldDisableControls }
 							onClick={ this.handleFinishClick }
 						>
-							Finish
+							{ translate( 'Finish', { context: 'verb' } ) }
```

Comment:
:disappointed: 68 existing translations will be lost with this change.
If the string must change, the following option has already been translated **63** times:
`translate( 'Finish', { context: 'spellcheck'} )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).